### ISSUE TITLE
Assembly version reported as 1.0.0.0

### ISSUE THREAD

**Issue by anpaz-msft** [Microsoft/jupyter-core#10]
For some unknown reason, the Jupyter core version reported on iqharp is 1.0.0.0
```
>dotnet iqsharp --version
Language kernel: 0.5.1902.2802
Jupyter core: 1.0.0.0
```

The version comes from the Assembly: https://github.com/Microsoft/jupyter-core/blob/master/src/Engines/BaseEngine.cs#L543

So, presumably it's a problem with the signed build.

**Comment by anpaz-msft** [comment]
On the build's definition, the `test` command that also builds the dll is lacking a version attribute:
https://github.com/Microsoft/jupyter-core/blob/d2a19c2354d8171cdda3a4672ec7c15d29d5eed5/ci/steps-build-core.yml#L11

We're only adding attribute to `pack`, but `pack` doesn't rebuild things. Need to add the parameter.

**Comment by cgranade** [comment]
Fixed by #14.
